README.md

```diff
@@ -11,3 +11,17 @@ Espera-se que todos os membros da comunidade sejam tratados com gentileza, não
 Aos que forem assediados, e aos que presenciem o assédio a outro membro da comunidade, espera-se que tentem interromper a agressão e que se reportem aos responsáveis pelo meio no qual a agressão ocorreu, ou diretamente a APyB.
 
 A Associação Python Brasil agradece a ajuda de todos para que possamos fomentar uma comunidade saudável e acolhedora. 
+
+# English
+
+This document is directed to entire community of Python programming language of Brazil. Hereby, the Associção Python Brasil (APyB - Python Brazil Association), had expressed expectations and guidelines to be followed in all the media, dissemination and use, whether online or offline, related to Python programming language.
+
+The purpose of this document is to protect everyone involved in the community for moral damages and dangers. The APyB dedicated to promoting a respectful and harassment-free community for all.
+
+In any of these environments harassment or intimidation should not be tolerated any form. Harassment includes offensive comments, verbal or electronic, related to characteristics or personal choices, as well as comments or sexual images in public or digital spaces, deliberated intimidation, bullying, photographs or recordings that generate embarrassment, inappropriate physical contact or unwanted sexual in any form. They are also condemned harassment of bad jokes from sexist nature, racist, or that offend / exclude any individual or group of individuals.
```

Comment:
- "in" any form
- unwanted sexual "actions"
- Harassment in the form of sexist or racist jokes  that offend/exclude any individual or group of individuals is also condemned.